CodeEdit/NavigatorSidebar/SourceControlNavigator/SourceControlSearchToolbar.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  SourceControlSearchToolbar.swift
+//  CodeEdit
+//
+//  Created by Nanashi Li on 2022/05/05.
+//
+
+import SwiftUI
+import CodeEditUI
+
+struct SourceControlSearchToolbar: View {
+
+    @Environment(\.colorScheme)
+    var colorScheme
+
+    @FocusState
+    private var isFocused: Bool
+
+    @Environment(\.controlActiveState)
+    private var controlActive
+
+    @State
+    private var text: String = ""
```

Comment:
nit: we don't have to specify that this is of type _String_ since we have to initalize it